### PR TITLE
DM-32756: Compute average FWHM of the PSF on an exposure as the media…

### PR DESCRIPTION
**Pull request by surhudm**
…n of PSF determined on a grid of points

### CODE REVIEW

File: python/lsst/afw/image/exposure/_exposureContinued.py [hunk 1]
```diff
@@ -31,6 +31,10 @@
 from ..image._fitsIoWithOptions import imageReadFitsWithOptions, exposureWriteFitsWithOptions
 from ._exposure import ExposureI, ExposureF, ExposureD, ExposureU, ExposureL
 from .exposureUtils import bbox_to_convex_polygon, bbox_contains_sky_coords
+from lsst.pex.exceptions import InvalidParameterError
+import lsst.geom as geom
+
+sigma2fwhm = 2.*np.sqrt(2.*np.log(2.))
```
Comment (ktlim) [inline python/lsst/afw/image/exposure/_exposureContinued.py:37]: Module variables used as module global constants are UPPERCASE_WITH_UNDERSCORES.  https://developer.lsst.io/python/style.html#naming-conventions-for-science-pipelines

Comment (surhudm) [reply python/lsst/afw/image/exposure/_exposureContinued.py]: Addressed in #f56a622f1

**Comment by ktlim** [comment]
Oh, also this really should have a test case added...

**Comment by arunkannawadi** [comment]
Changing the base to a corresponding ticket branch within `lsst` so we can test this more thoroughly and merge it to `main`. Thanks @surhudm !

**Pull request merged**